### DISCUSSION TITLE
[2주차][구현] 테스트 케이스 모음집

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/js-algorithm-basecamp#18] [General]
[2주차][구현] 테스트 케이스 모음집입니다.

**Comment by bigsaigon333** [comment]
### 문제 번호
4-2. 큰 수의 법칙
### Signature
`(N:number, M:number, K:number, nums: number[]) => number`

**Reply by bigsaigon333** [reply]
`(5, 8, 3, [2, 4, 5, 4, 6]) => 46`

**Comment by Sunmon** [comment]
### 문제 번호

12-9. 문자열 압축

### Signature

`(s:string) => number`

### Conditions

- 1 <= s.length  <=1,000
- s는 알파벳 소문자로만 이루어져 있습니다.

**Reply by Sunmon** [reply]
`"a" => 1`

`"aa" => 2`
